src/aolayer.cpp

```diff
@@ -531,6 +531,8 @@ void AOLayer::kill()
   this->clear();
   movie_frames.clear();
   movie_delays.clear();
+  last_max_frames = max_frames;
+  max_frames = 0;
```

Comment:
I think this was the real bug - not setting max_frames to 0

Reply:
I'll try it out...

Reply:
Can confirm, you're right this was the bug